modules/persistent-storage-csi-vsphere-disable-storage.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-disable-storage_{context}"]
+= Disabling storage on vSphere
+
+CLuster administrators might want to disable the VMWare vSphere Container Storage Interface (CSI) Driver Operator as a day 2 operation, so the the vSphere CSI driver with its operator does not interface with your vSphere setup.
+
+The consequences of disabling and enabling storage on vSphere are described in the following table.
+
+.Consequences of disabling/enabling storage on vSphere
+|===
+|Disabling | Enabling
+
+a| * vSphere CSI Driver Operator un-installs the CSI driver.
+
+* Storage container orchestration (CO) should be healthy.
+
+* vSphere-problem-detector continues running, but does not emit alerts or events, and checks less frequently (once per 24 hours).
+
+* All existing persistent volumes (PVs), persistent volume claims (PVCs), storage classes, and vSphere storage policies are unchanged:
+
+** vSphere PVs cannot be used in new pods.
+
+** vSphere PVs stay mounted and attached forever to existing nodes for existing pods. These pods remain in terminating state indefinitely after deletion.
+
+* Storage classes can be removed
+
+|* vSphere CSI Driver Operator re-installs the CSI driver. 
+
+* If necessary, the vSphere CSI Driver Operator creates the vSphere storage policy.
+
+* The number of clusters that have opted out are reported in a new metric.
+|===
```

Comment:
Table has both bullets and asterisks. Maybe stick with one or the other for consistency sake?